addons/web/static/tests/views/list_tests.js

```diff
@@ -861,6 +881,7 @@ QUnit.module('Views', {
         assert.ok(form.$('tbody tr:eq(2) td input').val(),
             "Value 2 should be third (shouldn't be sorted)");
 
+        form.$('.o_form_sheet_bg').click();
```

Comment:
?

Reply:
When list view is editable (active editing), We've now prevented sorting. So to sort, Now I click on this element to focus out from active editing.

Reply:
makes sense :) Thanks